### PR TITLE
test core 1a

### PR DESCRIPTION
**Pull request by firelegacy0**

### CODE REVIEW

File: enjin-core/index.js [hunk 1]
```diff
@@ -1 +1,2 @@
-console.log("Hello World")
\ No newline at end of file
+console.log("Hello World")
```
Comment (waterlegacy0) [inline enjin-core/index.js:1]:
```suggestion
console.log("Hello Worl1d")
```

**Review by waterlegacy0 — CHANGES_REQUESTED** [review]
Test Fix

File: enjin-core/index.js (hunk 1 above)
Comment (firelegacy0) [reply enjin-core/index.js:1]: test comments

Comment (firelegacy0) [reply enjin-core/index.js:1]: test comment 2

**Review by waterlegacy0 — DISMISSED** [review]
1

**Comment by firelegacy0** [comment]
test

File: enjin-core/index.js (hunk 1 above)
Comment (waterlegacy0) [reply enjin-core/index.js:1]: test

**Review by waterlegacy0 — DISMISSED** [review]

File: enjin-core/index.js (hunk 1 above)
Comment (firelegacy0) [reply enjin-core/index.js:1]: test 3

Comment (firelegacy0) [reply enjin-core/index.js:1]: 1

**Review by waterlegacy0 — APPROVED** [review]

**Review by waterlegacy0 — CHANGES_REQUESTED** [review]
test hold

**Review by waterlegacy0 — APPROVED** [review]

**Review by waterlegacy0 — CHANGES_REQUESTED** [review]
test4

**Review by waterlegacy0 — APPROVED** [review]

File: enjin-core/index.js (hunk 1 above)
Comment (waterlegacy0) [reply enjin-core/index.js:1]: no

**Review by waterlegacy0 — CHANGES_REQUESTED** [review]

**Review by waterlegacy0 — APPROVED** [review]

**Review by waterlegacy0 — DISMISSED** [review]
hold

**Review by waterlegacy0 — APPROVED** [review]
huh

**Review by waterlegacy0 — DISMISSED** [review]
hold pls

**Review by waterlegacy0 — APPROVED** [review]

**Review by waterlegacy0 — DISMISSED** [review]
no

**Review by waterlegacy0 — CHANGES_REQUESTED** [review]
test hold again

**Review by waterlegacy0 — APPROVED** [review]

**Review by waterlegacy0 — DISMISSED** [review]
no

File: enjin-core/index.js (hunk 1 above)
Comment (firelegacy0) [reply enjin-core/index.js:1]: test existing trigger

Comment (firelegacy0) [reply enjin-core/index.js:1]: edge case test

Comment (firelegacy0) [reply enjin-core/index.js:1]: huh

Comment (firelegacy0) [reply enjin-core/index.js:1]: test fail

Comment (firelegacy0) [reply enjin-core/index.js:1]: 1

Comment (firelegacy0) [reply enjin-core/index.js:1]: test
